hoomd/hpmc/tune/move_size.py

```diff
@@ -350,6 +394,12 @@ def scale_solver(cls,
                 considered tuned. The tolerance should not be too much lower
                 than the default of 0.01 as acceptance rates can vary
                 significantly at typical tuning rates.
+
+        Note:
+            Increasing ``gamma`` towards 1 does not necessarily speed up
+            convergence and can slow it down. In addition, large values of
+            ``gamma`` can make the solver unstable, especially when tuning
+            frequently.
```

Comment:
```suggestion
```
I don't think this is as true as in the secant solver. This is a simple scaling, and is more stable albeit slower than the secant slower in many use cases (as far as I could tell).